CrmManager.php

```diff
@@ -41,8 +27,6 @@ public function __construct(array $settings)
      */
     public function sendPerson(array $clientEntity): int
     {
-        $this->client->setCredentials($this->settings);
```

Comment:
Encapsulation principle violation. Context class doesn't have to know about internal logic realization